op-batcher/batcher/channel_manager.go

```diff
@@ -70,14 +71,18 @@ func (s *channelManager) Clear(l1OriginLastClosedChannel eth.BlockID) {
 	defer s.mu.Unlock()
 	s.log.Trace("clearing channel manager state")
 	s.blocks.Clear()
+	s.blockCursor = 0
```

Comment:
`blockCursor` should already have zero value by default, this is not needed

Reply:
I agree that is the default value, but it may have changed during the course of the program. To be clear, this is not a constructor but a resetter.